assets/src/js/header.js

```diff
@@ -132,7 +133,10 @@ const setMobileTabsMenuScroll = () => {
   let supportsPassive = false;
   try {
     const opts = Object.defineProperty({}, 'passive', {
-      get: function() { supportsPassive = true; } // eslint-disable-line getter-return
+      get() {
```

Comment:
I think that here it should be:
```
get: () => {
   supportsPassive = true;
}
```

Reply:
I still need to add the return statement for Sonar Cloud issues

<img width="823" alt="Screenshot 2023-03-30 at 10 15 55" src="https://user-images.githubusercontent.com/38656549/228789643-e7caa617-2685-4b08-b677-6ca22d6cbd60.png">

Reply:
I see that @lithrel initially added this function, I'd like his input here since I'm not sure I really understand what it does 😅

Reply:
(sorry, missed the point :grin: ) So it doesn't have to return anything, it just sets the `supportsPassive` value

Reply:
That is fine, the return statement I added is just for tests to pass and not complain about bugs.

Reply:
I know, but I just wanted to make sure that it doesn't affect the function's expected behaviour?

Reply:
@mleray just tested it, seems to work for me :+1: